packages/ui/src/components/_utils/analytics.ts

```diff
@@ -87,6 +92,8 @@ export function sendAnalyticsClickEvent<Component extends ComponentNames['click'
             component,
             ...data,
         }
```

Comment:
Wtf is container again?

Reply:
I'll remove comments to let you put proper description by yourself, since I'm not deep in purposes of bridge & UI packages